Dockerfile

```
@@ -0,0 +1,13 @@
+FROM node:10-alpine
+
+# install dependencies
+RUN apk add git
```

https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache

```suggestion
RUN apk add --no-cache git
```